src/main/java/net/sf/jabref/gui/EntryMarker.java

```diff
@@ -196,8 +196,6 @@ public static int isMarked(BibEntry be) {
     }
 
     public static boolean shouldMarkEntries() {
-        return (Globals.prefs.getBoolean(JabRefPreferences.MARK_IMPORTED_ENTRIES)
-                && (Globals.prefs.getBoolean(JabRefPreferences.USE_OWNER)
-                        || Globals.prefs.getBoolean(JabRefPreferences.USE_TIME_STAMP)));
+        return Globals.prefs.getBoolean(JabRefPreferences.MARK_IMPORTED_ENTRIES);
```

Comment:
how much effort would it be to create a test case? :innocent: 
- set marking in the prefs to true
- Import an entry
- check if the entry is marked